lib/commands/add-library.ts

```diff
@@ -0,0 +1,28 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import path = require("path");
+import Future = require("fibers/future");
```

Comment:
It depends how do you plan on using the API. It is available both as a constructor (i.e. new Future()) and as a function (i.e. future()). Import it the way you plan to use it. See https://github.com/laverdet/node-fibers#api-documentation